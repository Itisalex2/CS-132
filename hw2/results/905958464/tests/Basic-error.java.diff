1c1
< Program type checked successfully
---
> Type error
