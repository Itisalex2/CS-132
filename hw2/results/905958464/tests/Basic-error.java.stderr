Visiting main class. Added main class: ClassInfo{name='Main', fields={}, methods={main: MethodInfo{name='main', returnType='void', parameters={a: STRING_ARRAY}, localVariables={}}}}
Visiting class: ClassInfo{name='A', fields={}, methods={}}
Added method: run of type INT to class A
Added local variable: x of type INT to method run
Symbol Table:
Class: A
ClassInfo{name='A', fields={}, methods={run: MethodInfo{name='run', returnType='INT', parameters={}, localVariables={x: INT}}}}
Class: Main
ClassInfo{name='Main', fields={}, methods={main: MethodInfo{name='main', returnType='void', parameters={a: STRING_ARRAY}, localVariables={}}}}

